ide/gsf.testrunner/src/org/netbeans/modules/gsf/testrunner/TestCreatorProviderProcessor.java

```diff
@@ -47,11 +47,11 @@ protected boolean handleProcess(Set<? extends TypeElement> annotations, RoundEnv
             if(registration == null) {
                 continue;
             }
-            File f = layer(e).instanceFile("Services", null);
+            File f = layer(e).instanceFile("Services/TestCreatorProviders", null);
```

Comment:
Compatible, Services Lookup is recursive.

Reply:
Right. It is.